src/components/tab-group/tab-group.stories.tsx

```diff
@@ -0,0 +1,56 @@
+import type { Meta, StoryObj } from '@storybook/react';
+
+import { TabGroup } from './tab-group';
+
+const meta = {
+  title: 'Component/TabGroup',
+  component: TabGroup,
+  parameters: {},
+  tags: ['autodocs'],
+  argTypes: {},
+} satisfies Meta<typeof TabGroup>;
+
+export default meta;
+type Story = StoryObj<typeof meta>;
+
+export const OneItem: Story = {
+  args: {
+    tabs: [{ text: 'Tab 1', onClick: () => console.log('Tab 1 clicked') }],
+  },
+};
+export const TwoItem: Story = {
+  args: {
+    tabs: [
+      { text: 'Tab 1', onClick: () => console.log('Tab 1 clicked') },
```

Comment:
Hat es nicht geklappt mit actions?

Reply:
nein hab leider keine Lösung gefunden wie es genutzt werden kann wenn der onClick event in einem Object ist...